modules/planning/proto/open_space_roi_decider_config.proto

```diff
@@ -3,23 +3,30 @@ syntax = "proto2";
 package apollo.planning;
 
 message OpenSpaceRoiDeciderConfig{
-  // copy from message ROIConfig, planner_open_space_config.proto
+  // roi scenario definitions
+  enum RoiType {
+    NOT_DEFINED = 0;
+    PARKING = 1;
+    PULL_OVER = 2;
+    NARROW_STREET_U_TURN = 3;
+  }
+  optional RoiType roi_type = 1;
   // longitudinal range of parking roi
-  optional double roi_longitudinal_range = 1 [default = 10.0];
+  optional double roi_longitudinal_range = 2 [default = 10.0];
   // parking spot range detection threshold
-  optional double parking_start_range = 2 [default = 7.0];
+  optional double parking_start_range = 3 [default = 7.0];
```

Comment:
@JasonZhou404 Just add a TODO here to remind ourselves that this might need to either merge or differentiate with the ones from senarios. (I.e, PullOver), as of now we set similar things twice in Scenarios level and in Core algorithm level.